solutions/cses-1091.mdx

```diff
@@ -51,3 +54,124 @@ int main(){
 	return 0;
 }
 ```
+
+</CPPSection>
+
+<JavaSection>
+```java
+import java.io.*;
+import java.util.*;
+
+public class ConcertTickets {
+	public static void main(String[] args) throws IOException {
```

Comment:
is `throws IOException` needed?

Reply:
yeah I think so, `nextInt()` throws IOException